koap/build.gradle.kts

```diff
@@ -15,7 +15,7 @@ jacoco {
 
 kotlin {
     jvm()
-    js(BOTH).browser()
+    js().browser()
```

Comment:
Doesn't this then default to `LEGACY`? Or did that behavior change with the latest compiler?

I guess the real question is what is the default value of `kotlin.js.compiler`?

Reply:
I assumed IR was the default, but I'll set `kotlin.js.compiler` explicitly to be sure. 👍

Reply:
It appears you were right. After setting it explicitly I'm not getting the GitHub pages to build. Investigating...

Reply:
Fixed. Apparently the distribution directory changed between the compilers.